lib/upfluence/bank/currency_updater.rb

```diff
@@ -0,0 +1,19 @@
+module Upfluence
+  module Bank
+    class CurrencyUpdater
```

Comment:
if you do something that generic (that as nothing to do with Currency Update) maybe keep the name  generic as well

Reply:
I have hesitated, you're right :+1: